web/src/main/java/no/nav/foreldrepenger/web/server/jetty/JettyServer.java

```diff
@@ -65,7 +65,10 @@ protected JettyServer(int serverPort) {
     protected void bootStrap() throws Exception {
         konfigurerSikkerhet();
 
-        settJdniOppslag(DatasourceUtil.createDatasource("defaultDS", 30));
+        // Sett System.setProperty("task.manager.runner.threads", 10); til å endre antal prosesstask tråder. Default 10.
+        // `maxPoolSize` bør være satt minst til verdien av `task.manager.runner.threads` + 1 + antall connections man ønsker.
+        // `minIdle` sørger for at det alltid er en tilkobling klar når alle 10 prosesstask-trådene samtidig trenger en connection.
+        settJdniOppslag(DatasourceUtil.createDatasource("defaultDS", 25, 10));
```

Comment:
Hm, fpsak har default threads for TaskMgr = 10 (+1 for polling) 
Jetty har en threadpool på min = 8 og max = 200 - siden vi ikke angir noen annen verdi - ved trege kall videre så kan innkommende requests blokke alle connections slik at TaskMgr blir starved (noen sekund). Sjelden dette er et  faktisk problem. 
Noen spesiell grunn til å sette ned max fra 30 til 25?

Reply:
Tingen er at det skjer alltid rundt kl 7:00 når prosesstaskene slår til. En mulighet er at connection timeout er for lav - den var tidligere på 30 sekunder.

Nei, ikke noe spesiell grunn til å sett maks til 25, kan sette den opp igjen.

Reply:
Aner ikke om 20, 25, 30, 50 er optimalt. 
Burde kanskje se på Prometheus-tall fra Hikari på pool-utnyttelse pr app - ref DatasourceUtil - metrics
Men det haster ikke nå.

Reply:
Vi har nok ikke 2 milioner samtidige brukere ;) Vi tar det senere.